app/javascript/react/components/RestaurantShow.js

```diff
@@ -23,11 +25,34 @@ const RestaurantShow = (props) => {
     .catch(error => console.error(`Error in fetch: ${error.message}`))
     }, [])
 
-  return (
-      <h1>
-        {restaurant.name}
-      </h1>
-  )
+    const reviewTileArray  = () => restaurant.reviews.map((review) => {
+      
+      return(
+        <ReviewTile
+          key={review.id}
+          rating={review.rating}
+          partySize={review.party_size}
+          masksEmployees={review.masks_employees}
+          masksCustomers={review.masks_customers}
+          socialDistancing={review.social_distancing}
+        />
+      )
+    }) 
+  
+    return(
+      <div>
+        <h2>
+          {restaurant.name}
+        </h2>
+        <p>
+          Cuisine Type: {restaurant.cuisine}
+        </p>
+        <ReviewForm id={id}/>
+        <ul>
+          {Object.keys(restaurant).length !== 0 ? reviewTileArray() : null}
```

Comment:
where possible we should try to avoid handling logic inside the return of a component, this check could be done before mapping your review tiles